Source/WebCore/html/canvas/GPUCanvasContextCocoa.h

```diff
@@ -77,7 +77,7 @@ class GPUCanvasContextCocoa final : public GPUCanvasContext {
     bool isWebGPU() const override { return true; }
 
 private:
-    explicit GPUCanvasContextCocoa(CanvasBase&, GPU&);
+    explicit GPUCanvasContextCocoa(CanvasBase&, Ref<GPUCompositorIntegration>, Ref<GPUPresentationContext>);
```

Comment:
`Ref<GPUCompositorIntegration>&&, Ref<GPUPresentationContext>&&`. We don't generally pass smart pointers by value in WebKit.